packages/story-editor/src/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -56,9 +56,10 @@ describe('Right Click Menu integration', () => {
   async function closeRightClickMenu() {
     const framesLayer = fixture.screen.getByTestId('FramesLayer');
     // close browser default (only shows in puppeteer tests)
-    await fixture.events.click(framesLayer);
+    const rect = framesLayer.getBoundingClientRect();
+    await fixture.events.mouse.click(rect.left + 1, rect.top + 1);
     // close right click menu
-    await fixture.events.click(framesLayer);
+    await fixture.events.mouse.click(rect.left + 1, rect.top + 1);
```

Comment:
This function was clicking the right click menu in one test. Changed it so that we always click the background.

Once #8679 is merged, then clicking anywhere will work as long as it's not on the menu directly.